app/controllers/organisers_registration_controller.rb

```diff
@@ -0,0 +1,55 @@
+class OrganisersRegistrationController < ApplicationController
+ before_action :require_organiser
+ def create
+    @user = User.new(email: params[:email])
```

Comment:
right sorry you can remove hidden_field form organiser form and just call `@user = User.new(email: params[:email], role: 1)` as we want to call this method only when creating organiser so we can do that :) And obviously current line 5 should be removed.

Btw why did you move create to this controller?